MCD-182: (Developer Mode) Auto Correct

yahyayyash opened the pull request:
User could tap Polo 10 times on the game scene to enable auto correct notes.

Since developer mode isn't available yet, additional condition need to be added in the future.

yahyayyash committed:
MCD-182: (Developer Mode) Auto Correct

User could tap Polo 10 times on the game scene to enable auto correct notes.

Since developer mode isn't available yet, additional condition need to be added in the future.

adhxll reviewed (changes requested):
Overall review:
-
1. Function CountButtonClick() di GSUIManager
 => Penamaannya sangat general! Boleh banget dikasih bau ala” developer sih misal OnTappedDevButtonCount ?? (idk tho WKKW) 
 => I think generally it’s okay, but sepertinya butuh countdown untuk reset buttonCounter-nya jadi 0 lagi ketika developer gajadi nge tap 10 kali, buttonCounternya bakal reset sendiri ke 0. 
2. Enum DeveloperMode: 
 => boleh ditaruh di file baru di folder Script > Utilities > Enum
  => Thank you btw uda dibuatin semua mantab
   => Nanti coba digabung sama yang punya aqshal, soale dia bikin 1 playerpref buat indikasi devmode aktif/engga 

yahyayyash committed:
MCD-182: (Adjustment) Add Timer on Tapped Event

- Implement timer on tapped event (when Polo is tapped to enable developer mode), buttonCount will reset every 5s.
- Move DeveloperMode enum from SettingList to enum folder.
